src/datasets/load.py

```diff
@@ -161,7 +161,20 @@ def get_imports(file_path: str):
 
     logger.info("Checking %s for additional imports.", file_path)
     imports = []
+    is_in_docstring = False
```

Comment:
@lhoestq, would be nice if you can take a look! I think this is pretty save for now. The regex only hits if there is exactly one occurrence of `"""` to switch "in_docstring" mode to True

Reply:
yes this looks good :)